Can't resolve 'react-icons/lib/md/code' in code-input

There seems to be an unmet dependency in `code-input` causing it to fail

```Failed to compile.

Error in ./node_modules/@sanity/code-input/lib/schema.js
Module not found: Error: Can't resolve 'react-icons/lib/md/code' in '<redacted>/sanity/node_modules/@sanity/code-input/lib'
 @ ./node_modules/@sanity/code-input/lib/schema.js 10:35-69
 @ ./node_modules/@sanity/code-input/lib/schema.js (all:part:@sanity/base/schema-type)
 @ ./schemas/schema.js (part:@sanity/base/schema)
 @ ./node_modules/@sanity/default-layout/lib/components/DefaultLayout.js
 @ ./node_modules/@sanity/default-layout/lib/components/DefaultLayoutContainer.js (part:@sanity/base/root)
 @ ./node_modules/@sanity/base/lib/components/SanityRoot.js (part:@sanity/base/sanity-root)
 @ ./node_modules/@sanity/server/lib/browser/entry-dev.js
 @ multi ./node_modules/react-hot-loader/patch.js ./node_modules/normalize.css/normalize.css ./node_modules/@sanity/server/lib/browser/entry-dev.js```

--- a/packages/@sanity/code-input/package.json
+++ b/packages/@sanity/code-input/package.json
@@ -21,7 +21,8 @@
   ],
   "dependencies": {
     "lodash": "^4.17.4",
-    "react-ace": "^5.0.1"
+    "react-ace": "^5.0.1",
+    "react-icons": "^2.2.7"
   },
   "devDependencies": {
     "@sanity/check": "0.140.3",

